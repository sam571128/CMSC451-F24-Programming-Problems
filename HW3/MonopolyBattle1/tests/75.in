33 791 526591882
3 18
22 3
32 2
33 29
20 19
13 4
32 21
19 1
1 7
13 24
17 33
10 13
20 6
29 20
20 4
27 33
18 33
5 14
17 27
9 3
14 4
5 25
31 26
2 16
5 27
2 21
15 25
2 2
14 28
10 9
19 18
10 2
14 19
30 8
23 29
28 13
13 12
14 27
29 21
18 14
12 19
27 9
6 19
28 16
2 17
33 15
33 17
7 24
11 8
16 31
29 13
29 18
12 8
27 21
1 23
33 27
26 5
30 29
15 5
19 4
2 27
18 11
12 32
31 29
29 24
11 12
20 28
16 20
4 9
3 23
22 23
14 23
12 27
22 21
7 8
7 4
16 24
18 28
22 25
17 9
7 7
10 5
28 24
15 32
9 7
31 6
13 23
32 17
9 16
19 10
23 14
17 31
20 23
32 14
14 32
33 18
13 20
1 28
32 16
17 20
19 6
12 11
15 14
10 1
1 3
29 26
23 15
16 17
30 25
24 33
32 26
30 13
16 11
2 5
13 22
5 10
17 15
9 19
31 31
30 20
18 13
4 29
4 26
28 2
14 14
30 15
26 17
18 21
3 10
5 26
5 16
15 1
9 18
4 8
8 12
25 9
14 24
13 11
30 7
31 2
17 4
26 20
7 13
21 33
26 24
12 3
12 24
29 15
5 5
17 12
26 2
33 26
3 31
1 16
17 14
13 26
27 22
31 33
18 20
28 25
21 5
5 3
15 20
24 6
5 8
11 33
16 30
25 14
10 10
32 24
18 17
13 33
31 25
30 3
20 13
16 27
32 20
26 12
18 19
7 27
14 29
30 32
32 27
13 27
26 23
19 20
2 25
25 33
4 32
27 2
1 32
15 33
23 25
30 4
4 23
22 31
4 16
14 8
27 14
29 3
3 3
8 9
25 5
21 10
13 15
12 21
14 3
21 14
16 18
23 19
2 30
10 24
18 12
20 32
30 21
11 28
11 29
29 5
32 11
4 27
25 31
20 7
3 25
21 8
25 12
18 18
16 26
5 17
29 22
21 25
8 2
33 4
2 9
17 7
14 30
17 18
17 11
13 10
4 6
3 16
19 24
20 31
24 17
10 33
13 21
2 8
6 29
12 5
7 30
22 16
1 10
20 1
24 22
26 28
19 17
25 20
18 9
9 11
8 4
1 17
6 14
32 19
3 9
29 11
10 26
28 20
21 21
23 28
24 15
28 10
9 10
18 29
9 28
28 15
14 1
28 29
7 23
15 16
23 13
18 26
14 10
31 18
17 22
2 12
23 30
31 10
24 27
23 4
33 19
18 25
8 6
1 33
2 11
29 6
5 28
9 4
20 5
2 18
5 12
7 5
29 16
2 15
6 5
9 26
21 1
29 30
4 5
27 24
26 33
23 27
19 19
4 14
12 18
21 26
23 20
22 32
23 26
12 2
7 25
22 22
14 26
28 5
30 18
5 23
24 25
15 3
8 5
28 7
6 24
6 11
30 12
27 5
7 21
29 2
24 21
16 14
8 11
10 27
8 26
17 26
4 2
23 10
6 8
17 28
17 10
6 33
26 14
5 19
10 25
21 11
18 5
20 26
20 11
15 23
25 15
27 16
1 21
5 22
25 8
33 2
19 30
13 29
2 26
3 5
25 27
3 19
3 6
33 32
8 13
32 5
22 8
32 18
11 9
16 13
6 21
17 24
12 20
12 7
20 33
16 15
25 10
32 32
3 21
4 15
14 5
26 1
7 19
15 4
1 22
30 10
4 31
1 8
11 31
1 24
26 22
31 9
7 28
23 6
30 17
26 19
21 27
11 23
18 3
33 25
10 17
7 16
25 7
11 10
11 27
31 15
32 31
26 13
25 18
24 13
31 3
31 1
23 21
24 16
3 24
10 15
8 8
26 11
8 3
20 18
27 3
7 31
23 32
2 32
11 18
15 10
22 30
13 17
25 25
18 15
10 4
27 10
31 5
17 8
15 31
21 24
29 17
25 3
25 16
5 18
31 23
4 30
33 31
7 26
13 1
6 1
1 12
10 31
10 8
27 19
27 26
21 18
14 7
31 27
33 11
1 6
16 6
18 7
21 19
33 7
18 24
21 22
14 22
6 25
28 4
11 25
31 30
9 12
13 13
27 20
27 32
8 21
17 29
14 12
21 23
21 13
28 3
13 19
32 13
27 13
12 28
4 20
10 11
7 29
25 13
10 29
19 29
24 11
12 12
13 8
4 12
12 25
10 14
3 14
22 12
9 30
19 26
19 9
9 14
28 21
8 24
6 9
3 2
19 31
2 6
6 3
30 1
1 15
6 23
7 1
28 17
6 13
19 15
15 19
21 31
33 28
6 30
21 17
1 31
11 13
31 17
30 30
22 11
26 10
2 29
31 4
7 18
4 22
4 18
19 28
12 14
14 11
9 20
12 16
5 9
21 4
11 2
4 4
1 5
28 22
26 31
24 2
19 27
8 31
22 20
11 14
3 7
26 26
20 2
6 16
27 17
9 15
25 28
23 11
5 32
10 21
11 11
24 9
17 13
7 32
25 29
17 21
8 20
26 7
16 23
32 9
4 33
18 1
33 13
21 28
32 25
23 16
29 4
15 21
12 17
24 5
1 1
3 11
15 12
32 1
28 1
12 31
33 9
17 6
14 20
16 32
9 22
19 22
8 1
14 9
13 32
13 25
33 33
18 16
6 12
28 14
18 2
19 5
24 8
2 22
2 24
20 24
25 30
13 3
23 23
4 13
18 6
29 31
11 21
33 3
7 15
33 8
9 1
14 31
10 20
27 1
15 26
21 29
33 6
32 23
21 30
23 3
6 4
11 24
3 33
1 30
5 13
26 15
22 7
12 30
31 7
11 3
3 29
8 16
12 23
28 19
6 6
4 1
31 12
20 8
4 3
18 27
3 8
16 21
9 23
13 16
24 1
11 6
13 2
16 16
16 2
8 28
5 15
12 33
25 22
7 22
10 18
11 17
19 23
26 6
19 21
9 29
17 23
7 11
4 21
27 12
12 4
10 12
23 2
30 14
9 13
20 10
16 7
23 1
18 30
16 3
18 31
26 16
29 12
6 28
4 25
28 8
24 24
33 22
24 26
9 25
31 21
21 16
33 10
21 9
21 2
17 1
20 3
15 2
2 20
12 26
33 20
10 3
4 28
29 7
16 1
14 15
9 21
13 14
5 20
20 27
3 4
13 28
32 33
27 6
8 30
20 14
9 24
24 20
6 26
28 30
3 28
14 21
14 2
24 12
9 17
5 6
17 5
26 3
8 29
13 18
6 27
3 30
30 24
3 15
19 12
33 5
31 11
26 21
29 19
26 25
29 9
22 17
22 4
2 23
13 6
15 22
1 4
31 28
11 30
21 6
30 11
27 28
28 11
32 4
30 2
28 9
29 1
22 28
6 22
7 9
14 16
22 19
9 31
22 33
21 7
28 12
26 30
16 12
20 22
7 3
5 30
23 18
14 33
9 2
12 29
8 7
32 28
24 28
32 3
29 10
13 9
20 20
2 28
6 18
9 8
29 25
4 10
25 32
12 6
10 30
3 1
