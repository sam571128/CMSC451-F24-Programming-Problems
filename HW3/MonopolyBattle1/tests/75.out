55 63 55 55 57 55 55 55 55 90 62 55 55 55 55 55 55 55 55 55 55 115 55 55 55 55 55 55 55 167 55 63 55 